tests/Rules/PHPUnit/ShouldCallParentMethodsRuleTest.php

```diff
@@ -0,0 +1,35 @@
+<?php declare(strict_types = 1);
+
+namespace PHPStan\Rules\PHPUnit;
+
+use PHPStan\Rules\Rule;
+
+/**
+ * @extends \PHPStan\Testing\RuleTestCase<ShouldCallParentMethodsRule>
+ */
+class ShouldCallParentMethodsRuleTest extends \PHPStan\Testing\RuleTestCase
+{
+
+	protected function getRule(): Rule
+	{
+		return new ShouldCallParentMethodsRule();
+	}
+
+	public function testRule(): void
+	{
+		$this->analyse([__DIR__ . '/data/missing-parent-method-calls.php'], [
+			[
+				'Missing call to parent::setUp() method.',
+				32,
+			],[
```

Comment:
This should be formatted the same way as the next item (open square bracket on the next line)